tdrs-backend/tdpservice/settings/common.py

```diff
@@ -114,44 +113,21 @@ class Common(Configuration):
     EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
 
     # Whether to use localstack in place of a live AWS S3 environment
-    USE_LOCALSTACK = bool(os.getenv("USE_LOCALSTACK", 0))
-
-    # AWS Access Keys
-    AWS_S3_ACCESS_KEY_ID = os.getenv("AWS_ACCESS_KEY")
-    AWS_S3_SECRET_ACCESS_KEY = os.getenv("AWS_SECRET_ACCESS_KEY")
-    AWS_STORAGE_BUCKET_NAME = os.getenv("AWS_BUCKET")
-    AWS_REGION_NAME = os.getenv("AWS_REGION_NAME")
+    USE_LOCALSTACK = bool(strtobool(os.getenv("USE_LOCALSTACK", "no")))
 
     # Those who will receive error notifications from django via email
     ADMINS = (("Admin1", "ADMIN_EMAIL_FIRST"), ("Admin2", "ADMIN_EMAIL_SECOND"))
-    if "VCAP_SERVICES" in os.environ:  # pragma: nocover
```

Comment:
Removed VCAP_SERVICES references from the common settings file since those will only be present in cloud.gov environments.